README.md

```diff
@@ -8,7 +8,7 @@ Person A should modify content here.
 
 ## Person B:
 
-Person B should modify content here.
+hello - raisah was here
```

Comment:
I might suggest writing a test for this line of code, :)

Reply:
Thanks.